Component/PetList/PetList.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import pets from "@/Lib/Data";
+import Pet from "../Pet/Pet";
+
+const StyledPetList = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function PetList() {
+  return pets.length > 0 ? (
+    <StyledPetList>
+      {pets.map((pet, index) => {
+        return <Pet key={index} petData={pet} />;
+      })}
+    </StyledPetList>
+  ) : (
+    <h1>The Pet is a lie</h1>
```

Comment:
Please use an appropriate message for the user here. 
Additionally, your app should have an `h1` element, no matter if your pet list is rendered or not. This could be the app name.